src/components/project-list-card.astro

```diff
@@ -33,9 +33,10 @@ const { data } = Astro.props;
 
 <style>
   .card {
+    width: 100%;
```

Comment:
I had a slightly different fix stashed up locally.

Not a suggestion per se, just so you can easily copy paste and try it locally.
```suggestion
      .card {
    background-color: hsl(var(--blue-base) 95%);
    display: grid;
    padding: 1.5rem;
    border-radius: var(--border-radius);
    gap: 1.5rem;
  }

  @media (min-width: 850px) {
    .card {
      grid-template-columns: 2fr 1fr;
    }
  }

  @media (max-width: 849px) {
    .card {
      grid-template-rows: auto;
    }
  }
```

Reply:
Fun little fact I learned while doing this: you don't need to write two media queries. you can make a default and then have the media query at the bottom. This makes it so the media query one is used when required, otherwise you use the default. 

in the suggestion you gave, we can have the `grid-template-rows` as the media query, and `grid-template-columns: 2fr 1fr;` as the default.

Reply:
Updated to use grid 💙